vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/CodeGen/Lkg/Microsoft.ReactNative.Managed.CodeGen.UnitTests.CodeGen.CodeGenMethodTests--DoubleArgCallbackParamReturnVoid--Sync.lkg

```diff
@@ -0,0 +1,5 @@
+moduleBuilder.AddSyncMethod("MyMethod", (global::Microsoft.ReactNative.IJSValueReader reader, global::Microsoft.ReactNative.IJSValueWriter writer) =>
```

Comment:
There are a lot of new LKGs which I had removed in my last PR. It seems that there was a wrong merge. Please remove all these new LKGs. The issue with them is that they generate wrong code: For synchronous methods they use resolve and reject callbacks which are not available for the sync methods.

Reply:
Thanks for removing them!